src/en/WuxiaDotBlog.lua

```diff
@@ -1,5 +1,5 @@
--- {"id":1376,"ver":"1.0.4","libVer":"1.0.0","author":"AriaMoradi"}
---- @author AriaMoradi
+-- {"id":1376,"ver":"1.0.5","libVer":"1.0.0","author":"AriaMoradi, Doomsdayrs"}
+--- @author AriaMoradi, Doomsdayrs
```

Comment:
1. The author field should reflect the single person who has contributed most to an extension (the *author*, not anyone fixing a small bug) imo.
2. The `---@author` line is not useful and should be removed.